### PR TITLE
Solidjs/add authors

### PR DESCRIPTION
**Pull request by hdJerry**
# Description
- P3: anyone who has authored part of this kit should add themself to the package.json

### CODE REVIEW

File: starters/solidjs-tailwind/package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "description": "SolidJS and TailwindCSS",
+  "author": "Maarten, Chinedu, Daina, Mattai, Mark, Victor, Dane, Jan, Jerry",
```
Comment (Megio) [inline starters/solidjs-tailwind/package.json:6]: Please change Mattai to Mattia 🤣 🙏🏻

Comment (mbicknese) [inline starters/solidjs-tailwind/package.json:6]: Hi @hdJerry, seeing the word "author" is singular and package.json offering another field called "contributors" would it make sense to switch to that one instead? I believe it takes an array so we can create an entry for everyone.

```json
{
   "contributors": [
     "Jerry Hogan <jerry.hogan@thisdot.co>",
     "Chinedu Okpala <chinedu.okpala@thisdot.co>",
     "Mattia \"Mattai\" Magi <mattia.magi@thisdot.co>,
     "..."
   ]
}
```

Comment (hdJerry) [reply starters/solidjs-tailwind/package.json:6]: > Please change Mattai to Mattia 🤣 🙏🏻

Why should i @Megio ? I prefer mine 🤣 . I agree with @mbicknese

Comment (hdJerry) [reply starters/solidjs-tailwind/package.json:6]: Okay, let me try this. Thanks

**Pull request merged**